.github/ISSUE_TEMPLATE/hackathon-project-template.md

```diff
@@ -1,9 +1,96 @@
 ---
 name: Brainhack Project Template
-about: Project submissions are now closed!
+about: This provides the backbone structure for your project description. Please follow along to submit your project!
 title: ''
 labels: 'Hackathon project'
 assignees: '@ohbm/project-monitors'
 
 ---
-**Project submissions for the 2020 OHBM Brainhack are now closed**
+**Guidelines**
+
+*We are very excited to meet you at the 2021 OHBM Brainhack 🎉* *To submit a project, you need to be an attendee of the 2021 OHBM Brainhack. We ask you to register first over [here](http://www.humanbrainmapping.org/HackathonReg/). Thank you!*
+
+*We have prepared a checklist to help with your project submission. Here is how to proceed:*
+ 1. *Before filling in any part, please submit this issue*
+ 2. *Check items in the checklist below as you go through them*
+ 3. *Once you are done (at least all 'required' items must be provided), please delete the "Guidelines" section add a comment saying 'hi @ohbm/project-monitors: My project is ready!'*
+
+Thank you!
+
+*After step 1 (issue submitted), we will assign a 'project monitor' to follow your submission. If at any time you need help or anything is unclear, please add a comment and ping your project monitor. Our team is here to help!*
+
+
+----------------------------
+## Project info
+<!-- *Please fill this in first and then submit the issue* -->
+
+**Title**:
+<!--Name of your awesome project. Please also update the title of the issue to be the title of your project-->
+
+**Project lead**:
+<!--Your name and GitHub login, possibly more than 1 lead-->
+
+**[Timezone](https://github.com/ohbm/hackathon2021/blob/master/.github/ISSUE_TEMPLATE/handbooks/projects.md#timezone)**:
+<!--UTC offset of your timezone (cf. https://www.timeanddate.com/time/map/ for example).-->
+
+**Description**:
+<!--Describe the main idea and context of your project in a few sentences.-->
+
+**Link to project**:
+
+**Mattermost handle**:
+
+**Goals for the OHBM Brainhack**
+
+**Good first issues**:
+
+**Skills**:
+
+**Chat channel**:
+<!-- If you are creating a channel on the [brainhack mattermost](https://mattermost.brainhack.org/) try to create a
+**public** channel with one of the following template names:
+
+- hbmhack-NAME_OF_YOUR_PROJECT
+- hbm-NAME_OF_YOUR_PROJECT
+
+These would be the corresponding URLs that you can paste here.
+
+https://mattermost.brainhack.org/brainhack/channels/hbmhack-NAME_OF_YOUR_PROJECT
+https://mattermost.brainhack.org/brainhack/channels/hbm-NAME_OF_YOUR_PROJECT
+-->
+
+<!--
+**Video channel**:
+
```

Comment:
As we have not finalised yet the platform, maybe we should keep this one but slightly rephrase the explanation?

Reply:
How about:
"We suggest that you create a "meeting space" on the selected platform..."

Reply:
Maybe we can substitute the lines from 65 to 68 saying "As we are not finalised the platform for the hackathon yet, for now there is no need to fill the section for video channel."